content/user/upgrading/multiple_php_versions.md

```diff
@@ -13,6 +13,12 @@ First, you should verify that the desired [PHP Version is available](/user/upgra
 
 Generally your host will only allow you to run one version of PHP per domain.  But if you create one or more subdomains, each subdomain may also have its own version of PHP.  
 
+Before proceeding, determine whether your hoster allows you to have multiple versions of PHP per domain (generally by using a `.htaccess` file).  If they do, you may just use that technique, and deploy your test site under `public_html` in a different folder than live, or you may follow these steps (for better separation of test and live), and simply use the `.htaccess` file to set the PHP version, in lieu of using MultiPHP or a similar tool. 
```

Comment:
> and deploy your test site under `public_html` in a different folder than live

Upon first read I interpreted it as "deploy your site under public_html".

Perhaps minor rewording something like: "and deploy your site to a different folder under public_html (different than your live site folder in the same location)"
... or something that clarifies slightly.
Not a huge issue, as one would eventually figure it out.

Reply:
Updating it now to make it even more clear with examples.   Thanks for the feedback.